scripts/download-core.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 set -o pipefail
 
-: ${REALM_CORE_VERSION:=0.96.2} # set to "current" to always use the current build
+: ${REALM_CORE_VERSION:=0.97.0} # set to "current" to always use the current build
```

Comment:
You'll want to skip core v0.97.0. It crashes on 32-bit iOS devices. v0.97.1 and newer have the fix.